midi/midi_test/libretro.c

```diff
@@ -190,7 +190,7 @@ void retro_get_system_av_info(struct retro_system_av_info *info)
    info->geometry.max_height   = HEIGHT;
    info->geometry.max_width    = WIDHT;
    info->timing.fps            = FPS;
-   info->timing.sample_rate    = 48000.0;
```

Comment:
I believe the midi test may need a sample rate.

Reply:
The MIDI test discards the sampled audio callbacks from the frontend, so I assumed there would be no need for a sample rate.